README.md

```diff
@@ -209,7 +209,7 @@ To actually start the robot driver use one of the existing launch files
 
     $ roslaunch ur_robot_driver <robot_type>_bringup.launch robot_ip:=192.168.56.101
 
-where **<robot_type>** is one of *ur3, ur5, ur10, ur3e, ur5e, ur10e, ur16e, ur20, ur30*. Note that in this example we
+where **<robot_type>** is one of *ur3, ur5, ur10, ur3e, ur5e, ur10e, ur16e, ur15, ur20, ur30*. Note that in this example we
```

Comment:
Not related to ur15 changes, but ur7e and ur12e seems to have been forgotten in the documentation. 
```suggestion
where **<robot_type>** is one of *ur3, ur5, ur10, ur3e, ur5e, ur7e, ur10e, ur12e, ur16e, ur15, ur20, ur30*. Note that in this example we
```